tests/e2e/util.go

```diff
@@ -1676,24 +1738,28 @@ func VsanObjIndentities(ctx context.Context, vs *vSphere) []string {
 		}
 		framework.Logf("Default cluster is choosen for test")
 	}
-	clusterComputeResource, vsanHealthClient, err := getClusterName(ctx, vs)
-	framework.Logf("clusterComputeResource %v", clusterComputeResource)
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	clusterComputeResource, vsanHealthClient = getClusterComputeResource(ctx, vs)
 
 	for _, cluster := range clusterComputeResource {
-		framework.Logf("cluster name %v", cluster.Name())
 		if strings.Contains(cluster.Name(), computeCluster) {
 			clusterConfig, err := vsanHealthClient.VsanQueryObjectIdentities(ctx, cluster.Reference())
+			framework.Logf("clusterconfig %v", clusterConfig)
```

Comment:
This print statement is not needed

Reply:
This will be helpful to analysis the test failure, hence I have added this log statement.